src/Features/Core/Portable/ExtractMethod/ExtractMethodOptions.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.CodeAnalysis.ExtractMethod
 {
     internal static class ExtractMethodOptions
     {
-        public static readonly PerLanguageOption<bool> AllowBestEffort = new PerLanguageOption<bool>(nameof(ExtractMethodOptions), nameof(AllowBestEffort), defaultValue: false,
+        public static readonly PerLanguageOption<bool> AllowBestEffort = new PerLanguageOption<bool>(nameof(ExtractMethodOptions), nameof(AllowBestEffort), defaultValue: true,
```

Comment:
are we okay with this? I wanted to turn this on, but had so many feedback saying no. telling me prove it is safe and test coverages and etc.

anyway, I am +1 on making this on. after this, we will not prevent people from extracting method on yield return and etc, also unbalanced return and etc.

Reply:
danger of turning on this is an exception on code generation. since it might try to extract code which code generation might not know how to handle.